src/generator.rs

```diff
@@ -32,8 +33,8 @@ pub enum Language {
     Python,
 }
 
-impl fmt::Display for Language {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+impl Display for Language {
+    fn fmt(&self, f: &mut Formatter<'_>) -> FmtResult {
```

Comment:
I'm on the fence about `s/fmt::Result/FmtResult`. I think `fmt::Result` is perhaps _slightly_ clearer given that you don't need to look up the alias.

(this is really picking at straws, but that's what this MR is about, right? :D)

Reply:
I think you are right! `fmt::Result` is short and very descriptive. 

And yes, every straw deserves to be picked :D